Python/heap.py

```diff
@@ -0,0 +1,71 @@
+# Implementation of heap from list in Python
+
+class Heap(object):
+
+    def __init__(self):
+        self.list = []
+        self.last_index = -1
+
+    def insert(self, x):
+        self.last_index += 1
+        if not self.list:
+            self.list.append(x)
+            return
+        self.list.append(x)
+        index = self.last_index
+        parent = int((index-1)/2)
+        while (self.list[index] > self.list[parent]) and parent >= 0:
```

Comment:
nit: line between while and the preceding line for readability.

needs a comment explaining whats going on too